.woodpecker/.feature.yml

```diff
@@ -1,10 +1,15 @@
 pipeline:
   build-and-push:
-    image: plugins/docker
+    image: woodpeckerci/plugin-docker-buildx
     settings:
+      platforms: linux/amd64,linux/arm64
       repo: "${CI_REPO_OWNER}/${CI_REPO_NAME%%-service}"
       tags: "feature-${CI_COMMIT_BRANCH##feature/}"
+      username: 
+        from_secret: docker_username
+      password: 
+        from_secret: docker_password
```

Comment:
I don't think this part is needed? According to the docs you can either use `secrets: [..]` or the `from_secret` setup you added here. So with this change we're essentially adding both.

It seems that the [`woodpeckerci/plugin-docker-buildx` looks for](https://codeberg.org/woodpecker-plugins/docker-buildx/src/commit/37718ded77131706c6c566b9c3817d5d9e35b7d5/cmd/docker-buildx/config.go#L226) either the `DOCKER_USERNAME` (which `secrets:` adds) or `PLUGIN_USERNAME` (which `from_secret` adds) environment variables, so either of these 2 setups should do.